### PR TITLE
Change text and link to the current repository name

### LINKED ISSUE #120: Change links and text to the current repository name
## Description
As a contributor,
I need all the links and text that is using `open-source-programs-web` changed into `anitab-forms-web`,
so that there is no confusion and 404 pages for newcomers.

### Update [Required]
- [ ] Change `open-source-programs-backend` to `anitab-forms-backend`.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3 hours.

### PR DESCRIPTION
**Pull request by untari**
### Description

I have changed the text and link in the README.md file according to the current repository name from  ```open-source-programs-web``` to ```anitab-forms-web``` and from ```open-source-programs-backend``` to ```anitab-forms-backend``` .

Fixes #120

### Type of Change:

- Documentation

### How Has This Been Tested?
- Clone the new repository name and to start the server do ``` cd anitab-forms-web ```
- Click to the updated link

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

### CODE REVIEW

**Review by codesankalp — CHANGES_REQUESTED** [review]
@untari Nice efforts. 
But can you please consider other files also like inside `docs/` and some part of code also?

**Comment by untari** [comment]
> @untari Nice efforts.
> But can you please consider other files also like inside `docs/` and some part of code also?
@codesankalp sure

**Comment by untari** [comment]
Hi, I'm trying to update the file in `docs` but apparently  my fork repository hasn't updated with the new file from the main repository. Any solutions how can I update my fork ? 
I tried to do upstream but it  didn't work. Thanks

**Comment by untari** [comment]
hi @codesankalp  I already updated text and links in `docs` and checked if there's any more file need to be change. let me know if I missed other files. Thanks

File: README.md [hunk 1]
```diff
@@ -37,7 +37,10 @@ npm start
 
 ## Contributing
 
-Please read the [Contributing guidelines](.github/CONTRIBUTING.md), [Code of Conduct](https://github.com/anitab-org/open-source-programs-web/blob/develop/CODE_OF_CONDUCT.md) and [Reporting Guidelines](https://github.com/anitab-org/open-source-programs-web/blob/develop/REPORTING_GUIDELINES.md)
+Please read the Contibuting guidelines, [Code of Conduct](https://github.com/anitab-org/anitab-forms-web/blob/develop/CODE_OF_CONDUCT.md) and [Reporting Guidelines](https://github.com/anitab-org/anitab-forms-web/blob/develop/REPORTING_GUIDELINES.md)
+
+
+Please read the [Contributing guidelines](.github/CONTRIBUTING.md), [Code of Conduct](https://github.com/anitab-org/anitab-forms-web/blob/develop/CODE_OF_CONDUCT.md) and [Reporting Guidelines](https://github.com/anitab-org/anitab-forms-web/blob/develop/REPORTING_GUIDELINES.md)
```
Comment (codesankalp) [inline README.md:40]: This thing is repeated twice. Please remove one.

**Review by codesankalp — CHANGES_REQUESTED** [review]
Almost done @untari, I have mentioned a change please look into that.

File: README.md [hunk 2]
```diff
@@ -39,9 +39,6 @@ npm start
 
 Please read the Contibuting guidelines, [Code of Conduct](https://github.com/anitab-org/anitab-forms-web/blob/develop/CODE_OF_CONDUCT.md) and [Reporting Guidelines](https://github.com/anitab-org/anitab-forms-web/blob/develop/REPORTING_GUIDELINES.md)
```
Comment (codesankalp) [inline README.md]:
```suggestion
Please read the [Contributing guidelines](.github/CONTRIBUTING.md), [Code of Conduct](https://github.com/anitab-org/anitab-forms-web/blob/develop/CODE_OF_CONDUCT.md) and [Reporting Guidelines](https://github.com/anitab-org/anitab-forms-web/blob/develop/REPORTING_GUIDELINES.md)
```

**Review by codesankalp — CHANGES_REQUESTED** [review]

**Review by codesankalp — DISMISSED** [review]
LGTM @untari :tada:

File: docs/docusaurus.config.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ module.exports = {
             {
               label: 'Backend Docs',
               href:
-                'https://github.com/anitab-org/open-source-programs-backend/wiki',
+                'https://github.com/anitab-org/oanitab-forms-backend/wiki',
```
Comment (Aaishpra) [inline docs/docusaurus.config.js]: Just a little mistake here.

**Review by Aaishpra — CHANGES_REQUESTED** [review]
Hi @untari thanks for the great detailed work here, just a typo in the link is there. Can you fix it then this will be ready to merge

File: docs/docusaurus.config.js (hunk 3 above)
Comment (untari) [reply docs/docusaurus.config.js]: done

File: README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs)
 [![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
 ![Open Source Love](https://img.shields.io/badge/Open%20%E2%9D%A4%EF%B8%8FSource-blue)
-[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/open-source-programs-web)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/anitab-forms-web)
 ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)
 
 # Open Source Programs (Frontend)
```
Comment (Aaishpra) [inline README.md]: We absolutely missed the heading can you also change this @untari

Comment (Aaishpra) [reply README.md]:
```suggestion
# AnitaB-Forms (Frontend)
```

Comment (Aaishpra) [reply README.md]: And the about section in the readme is still containing instances of `osp` instead of `anitab-forms`

Comment (untari) [reply README.md]: done, did I do it right?

Comment (Aaishpra) [reply README.md]: Yesss!!

**Review by Aaishpra — APPROVED** [review]
Thank you so much for the quick response @untari , Great work here 💯

**Review by codesankalp — APPROVED** [review]

**Comment by Aaishpra** [comment]
@isabelcosta need your review here.

**Review by isabelcosta — APPROVED** [review]
Thank you for your contribution @untari 🎉

**Comment by isabelcosta** [comment]
not sure why the Lint check is failing 🤔  https://github.com/anitab-org/anitab-forms-web/pull/134/checks?check_run_id=2530196220
I will merge this because I know it's not to do with the markdown updates to the code

**Pull request merged**